src/PublishPort.jsx

```diff
@@ -7,13 +8,51 @@ import { TextInput } from "@patternfly/react-core/dist/esm/components/TextInput"
 import { Popover } from "@patternfly/react-core/dist/esm/components/Popover";
 import { OutlinedQuestionCircleIcon, TrashIcon } from '@patternfly/react-icons';
 import cockpit from 'cockpit';
+import ipaddr from "ipaddr.js";
+
+import { FormHelper } from "cockpit-components-form-helper.jsx";
+import * as utils from './util.js';
 
 const _ = cockpit.gettext;
 
-export const PublishPort = ({ id, item, onChange, idx, removeitem, itemCount }) =>
+export function validatePublishPort(value, key, validationFailed) {
+    const delta = { ...validationFailed };
+    delete delta[key];
+
+    switch (key) {
+    case "IP":
```

Comment:
eslint expects the indentation this way, otherwise it'll complain

```
✘ [ERROR] ESLint errors found [plugin eslintPlugin]

change detected: .PublishPort.jsx.swp

/home/skobyda/Work/cockpit-podman/src/PublishPort.jsx
  25:1  error  Expected indentation of 4 spaces but found 8    indent
  26:1  error  Expected indentation of 8 spaces but found 12   indent
  27:1  error  Expected indentation of 12 spaces but found 16  indent
  28:1  error  Expected indentation of 8 spaces but found 12   indent
  29:1  error  Expected indentation of 4 spaces but found 8    indent
  30:1  error  Expected indentation of 8 spaces but found 12   indent
  31:1  error  Expected indentation of 12 spaces but found 16  indent
  32:1  error  Expected indentation of 12 spaces but found 16  indent
  33:1  error  Expected indentation of 16 spaces but found 20  indent
  34:1  error  Expected indentation of 8 spaces but found 12   indent
  36:1  error  Expected indentation of 8 spaces but found 12   indent
  37:1  error  Expected indentation of 4 spaces but found 8    indent
  38:1  error  Expected indentation of 4 spaces but found 8    indent
  39:1  error  Expected indentation of 8 spaces but found 12   indent
  40:1  error  Expected indentation of 12 spaces but found 16  indent
  42:1  error  Expected indentation of 8 spaces but found 12   indent
  43:1  error  Expected indentation of 8 spaces but found 12   indent
  44:1  error  Expected indentation of 12 spaces but found 16  indent
  46:1  error  Expected indentation of 8 spaces but found 12   indent
  47:1  error  Expected indentation of 4 spaces but found 8    indent
  48:1  error  Expected indentation of 4 spaces but found 8    indent
  49:1  error  Expected indentation of 8 spaces but found 12   indent
```

Reply:
Weird .. And it doesn't complain about two `}` at the same indentation level? :exploding_head: Anyway, let's ignore indeed.